templates/plugin-gruntfile.mustache

```diff
@@ -11,10 +11,11 @@ module.exports = function( grunt ) {
 			options: {
 				textdomain: '{{textdomain}}',
 			},
-			target: {
-				files: {
-					src: [ '*.php', '**/*.php', '!node_modules/**', '!php-tests/**', '!bin/**' ]
-				}
+			update_all_domains: {
```

Comment:
> It seems like a reasonable assumption to me that all textdomains are the same for a given project.

There are some cases where textdomains can be different(like textdomains for the back-end and textdomains for the front-end) - I think WooCommerce does it this way. However, since this is meant to be a boilerplate it should be fine(as in it shouldn't do everything out of the box).

If you think this change is reasonable, go ahead and merge it :smiley: